javascript-source/handlers/subscribeHandler.js

```diff
@@ -383,12 +485,27 @@
             $.say($.whisperPrefix(sender) + $.lang.get('subscribehandler.giftsub.msg.set'));
         }
 
+        /*
+         * @commandpath giftanonsubmessage[message] - Set a message
```

Comment:
Update spacing for parameter, and make the entire thing one line and update to message to `Set a message for anonymous gifting alerts.`